tools/goseq_1_22_0/get_length_and_gc_content.r

```diff
@@ -0,0 +1,59 @@
+# originally by Devon Ryan, https://www.biostars.org/p/84467/
+
+options( show.error.messages=F, error = function () { cat( geterrmessage(), file=stderr() ); q( "no", 1, F ) } )
+
+# we need that to not crash galaxy with an UTF8 error on German LC settings.
```

Comment:
well, I like that I can use these scripts on the commandline as well, and the debugging / development is much easier this way.